### PR TITLE
[RTE] Storybook update for rich text edit box

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

Storybook update for rich text edit box. This PR can be merged only after beta release.

# Why

# How Tested

Storybook

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/controlsUtils.ts [hunk 1]
```diff
@@ -362,6 +362,7 @@ export const controlsToAdd = {
   token: { control: 'text', defaultValue: '', name: 'Valid token for user', type: { name: 'string', required: true } },
   typingUsers: { control: 'object', defaultValue: defaultTypingUsers, name: 'Typing users' },
   isCaptionsFeatureActive: { control: 'boolean', defaultValue: true, name: 'Is captions on' },
+  richTextEditor: { control: 'boolean', defaultValue: true, name: 'Enable rich tex editor' },
```
Comment (palatter) [inline packages/storybook/stories/controlsUtils.ts:365]:
```suggestion
  richTextEditor: { control: 'boolean', defaultValue: true, name: 'Enable rich text editor' },
```

File: packages/storybook/stories/MessageThread/MessageThread.stories.tsx [hunk 2]
```diff
@@ -192,6 +197,11 @@ const Docs: () => JSX.Element = () => {
         checkout the details about these components
         [here](https://microsoft.github.io/fluentui-contrib/react-chat/?path=/story/chat--default).
       </Description>
+      <Description>
+        The MessageThread component supports lazy loading for the rich text editor used for editing messages. This means
```
Comment (palatter) [inline packages/storybook/stories/MessageThread/MessageThread.stories.tsx:201]: Can we include a link to sideEffects doc?

https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

Comment (vhuseinova-msft) [reply packages/storybook/stories/MessageThread/MessageThread.stories.tsx:201]: Updated!

**Review by palatter — APPROVED** [review]

**Comment by palatter** [comment]
Also is the rich text editor on by default for the messageThread preview?

**Comment by palatter** [comment]
Should this banner be at the top of the page? Or on each section?

image

**Comment by vhuseinova-msft** [comment]
> Also is the rich text editor on by default for the messageThread preview?



> Should this banner be at the top of the page? Or on each section?
> 
> image

Fixed

**Review by palatter — APPROVED** [review]

**Review by AmyL219 — APPROVED** [review]
LGTM

**Pull request merged**